src/Modules/Admin/Commands.Admin.Test/AddPowerBIEncryptionKeyTests.cs

```diff
@@ -0,0 +1,172 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Linq;
+using System.Management.Automation;
+using Microsoft.PowerBI.Api.V2.Models;
+using Microsoft.PowerBI.Commands.Common.Test;
+using Microsoft.PowerBI.Commands.Profile.Test;
+using Microsoft.PowerBI.Common.Abstractions;
+using Microsoft.PowerBI.Common.Api;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+
+namespace Microsoft.PowerBI.Commands.Admin.Test
+{
+    [TestClass]
+    public class AddPowerBIEncryptionKeyTests
+    {
+        private static CmdletInfo AddPowerBIEncryptionKeyCmdletInfo => new CmdletInfo($"{AddPowerBIEncryptionKey.CmdletVerb}-{AddPowerBIEncryptionKey.CmdletName}", typeof(AddPowerBIEncryptionKey));
+
+        private static string MockName = "KeyName";
+        private static string MockKeyVaultKeyUri = "KeyVaultUri";
+        private static bool MockDefault = true;
+        private static bool MockActivate = true;
+
+        [TestMethod]
+        [TestCategory("Interactive")]
+        [TestCategory("SkipWhenLiveUnitTesting")] // Ignore for Live Unit Testing
+        public void EndToEndAddPowerBIEncryptionKey()
+        {
+            using (var ps = System.Management.Automation.PowerShell.Create())
+            {
+                // Arrange
+                ProfileTestUtilities.ConnectToPowerBI(ps, PowerBIEnvironmentType.OneBox);
```

Comment:
Remove the PowerBIEnvironmentType input, it won't compile on Release builds as OneBox is conditionalized out of the compile

Reply:
Done